src/accountsdb/download.zig

```diff
@@ -11,7 +11,9 @@ const ThreadSafeContactInfo = sig.gossip.data.ThreadSafeContactInfo;
 const GossipService = sig.gossip.GossipService;
 const Logger = sig.trace.Logger;
 
-const DOWNLOAD_PROGRESS_UPDATES_NS = 30 * std.time.ns_per_s;
+const assert = std.debug.assert;
```

Comment:
removed most - [1764858](https://github.com/Syndica/sig/pull/316/commits/1764858a8e688817da750f1e811960118c44e62b) - some the type alias makes sense because its used so much throughout the code